tests/api/v1/endpoints/test_drp_endpoints.py

```diff
@@ -0,0 +1,288 @@
+import json
+from typing import Callable
+from unittest import mock
+
+import jwt
+import pytest
+from sqlalchemy.orm import Session
+from starlette.testclient import TestClient
+
+from fidesops.api.v1.scope_registry import (
+    PRIVACY_REQUEST_READ,
+    STORAGE_CREATE_OR_UPDATE,
+)
+from fidesops.api.v1.urn_registry import (
+    V1_URL_PREFIX,
+    DRP_EXERCISE,
+    DRP_STATUS,
+)
+from fidesops.core.config import config
+
+from fidesops.models.privacy_request import (
+    PrivacyRequest,
+    PrivacyRequestStatus,
+)
+from fidesops.schemas.privacy_request import PrivacyRequestDRPStatus
+from fidesops.util.cache import get_drp_request_body_cache_key, get_identity_cache_key
+
+
+class TestCreateDrpPrivacyRequest:
+    @pytest.fixture(scope="function")
+    def url(self) -> str:
+        return V1_URL_PREFIX + DRP_EXERCISE
+
+    @mock.patch(
+        "fidesops.service.privacy_request.request_runner_service.PrivacyRequestRunner.submit"
+    )
+    def test_create_drp_privacy_request(
+        self,
+        run_access_request_mock,
+        url,
+        db,
+        api_client: TestClient,
+        policy_drp_action,
+        cache,
+    ):
+
+        config.security.DRP_JWT_SECRET = "secret"
```

Comment:
It's probably worth setting a default for this value that's overridden on deployment [in the same way we do with the root client](https://github.com/ethyca/fidesops/blob/main/fidesops.toml#L20-L21). That way we won't need to override it here?